ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,169 @@
+      var accessorySize: MSFListAccessoryVnextSize!
+
+      public init(accessoryStyle: MSFListAccessoryVnextStyle,
+                  accessorySize: MSFListAccessoryVnextSize) {
+          self.accessoryStyle = accessoryStyle
+          self.accessorySize = accessorySize
+          self.themeAware = true
+
+          didChangeAppearanceProxy()
+      }
+
+      @objc open func didChangeAppearanceProxy() {
+          backgroundColor = appearanceProxy.backgroundColor
+          borderColor = appearanceProxy.borderColor
+          borderSize = appearanceProxy.borderSize
+          padding = appearanceProxy.padding
+          textFont = appearanceProxy.textFont
+          subtitleFont = appearanceProxy.subtitleFont
+          leadingTextColor = appearanceProxy.textColor.leading
+          trailingTextColor = appearanceProxy.textColor.trailing
+
+          switch accessoryStyle {
+          case .iconOnly, .none:
+              leadingIconColor = appearanceProxy.iconColor.iconOnly
+              trailingIconColor = appearanceProxy.iconColor.iconOnly
+          case .withLabel:
+              leadingIconColor = appearanceProxy.iconColor.withLabel
+              trailingIconColor = appearanceProxy.iconColor.withLabel
+          case .disclosure:
+              trailingIconColor = appearanceProxy.iconColor.disclosure
+          }
+
+          switch accessorySize {
+          case .icon, .none:
+              leadingIconSize = appearanceProxy.iconSize.icon
+              trailingIconSize = appearanceProxy.iconSize.icon
+              interspace = appearanceProxy.interspace.icon
+          case .persona:
+              leadingIconSize = appearanceProxy.iconSize.persona
+          case .disclosure:
+              trailingIconSize = appearanceProxy.iconSize.disclosure
+              interspace = appearanceProxy.interspace.disclosure
+          case .button:
+              interspace = appearanceProxy.interspace.button
+          }
+      }
+  }
+
+  public struct MSFListView: View {
+      @ObservedObject var state: MSFListItem
+      @ObservedObject var tokens: MSFListTokens
+      var numCells: [MSFListItem]
+
+      public init(cells: [MSFListItem], style: MSFListAccessoryVnextStyle, size: MSFListAccessoryVnextSize) {
+          self.state = MSFListItem()
+          self.tokens = MSFListTokens(accessoryStyle: style, accessorySize: size)
+          self.numCells = cells
+      }
+
+      public var body: some View {
+          List(numCells) { cell in
+              HStack {
+                  Image(cell.leadingView ?? "")
+                      .resizable()
+                      .frame(width: tokens.leadingIconSize, height: tokens.leadingIconSize)
+                  VStack(alignment: .leading) {
+                      Text(cell.title)
+                          .font(Font(tokens.textFont))
+                          .foregroundColor(Color(tokens.leadingTextColor))
+                      Text(cell.subtitle ?? "")
```

Comment:
Enclosing this with an if statement will help you work on the alignment issues when there's not subtitle.